### PR TITLE
Chore: refine tests

### PR DESCRIPTION
**Pull request by erikjung**
Re: #1 

This adds some tests.

/CC @mrgerardorodriguez

### CODE REVIEW

**Comment by mrgerardorodriguez** [comment]
Cool! :+1:

File: gulpfile.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (lyzadanger) [inline gulpfile.js]: Is this because this file is in ES5?

**Comment by lyzadanger** [comment]
Cool. I should "steal" some of those `yeoman-assert` assertions for file content for my own purposes in the builder!

LGTM!

File: gulpfile.js (hunk 1 above)
Comment (erikjung) [inline gulpfile.js]: Yes. I'll update the Gulpfile later on and remove the flag.

**Pull request merged**